src/documents/IoCTestingFramework.md

```diff
@@ -333,25 +577,60 @@ fluid.defaults("fluid.tests.asyncTestTree", {
 });
 ```
 
+### `markupFixture` property supporting fixtures written against markup in the host document
```

Comment:
Agreed that splitting it is very confusing.   However, even just wrapping the first code example in the minimum boilerplate required to run it seems to fail with infusion 2.0.0.  Here's the code:

http://pastie.org/10983346

When run in node 6.9.1, It fails with "no tests were queued" with no console.log output.